app/models/user.rb

```diff
@@ -58,7 +59,7 @@ class User
   field :locked_at,       type: Time
 
   # Validations
-  validates :email, :name, presence: true
+  validates :name, presence: true
```

Comment:
Learn something new every day! I'd say let's follow your pattern on this, then. Thanks for documenting.